### DISCUSSION TITLE
Problem with bindings in automation trigger CREATE ROW

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6347] [Help] [answered]
I am creating a trigger that inserts a row in table B when a row is inserted in table A.

How can I get values of table A fields to bind them with table B fields ? I do not see any available bindings...

Even the {{ Current User.email }} seems to be not available, the returned value is NULL.

**Comment by melohagan** [comment, marked as answer]
Hi @NicolasFabre81 

In the automation screen, the available trigger rows do not individually appear, but you can still reference them, e.g. `trigger.rows.ColumnNameOnTableA`

As for the *Current User* bindings, they are not available in automations because automations are run in the background automatically - there is no logged in user from that perspective.

**Reply by NicolasFabre81** [reply]
Ok thanks you're too fast !!
